tests/foreman/api/test_repositories.py

```diff
@@ -226,7 +226,7 @@ def test_positive_sync_upstream_repo_with_zst_compression(
 
 @pytest.mark.tier1
 @pytest.mark.manifester
-def test_negative_upload_expired_manifest(module_org, target_sat):
+def test_negative_upload_expired_manifest(request, default_org, target_sat):
```

Comment:
why using default_org instead of module_org?

Reply:
module scope org already has manifestest imported in it, and can't import new one, rather creating new org with function_org fixture I am decided to use default.

Reply:
I would rather recommend to create a new one than using the `default_org`. The reason behind is that any other test on the same SAT / x-dist worker (even from another test module) can upload a manifest to the `default_org` and make your test randomly fail this way and it will be difficult to debug which one it was.